docs/source/installation/plugins.rst

```diff
@@ -6,7 +6,7 @@ Plugins
 
 Plugins are features that extend Manim's core functionality. Since Manim is
 extensible and not everything belongs in its core, we'll go over how to
-install, use, and create your own plugins.
+install, use, and create your plugins.
```

Comment:
I prefer the original

Reply:
Well your own is the same your cause it refers to the person’s plugins. So it is better to cut off own imo